docs/components/GeneratedPropTable.js

```diff
@@ -112,7 +117,6 @@ export default function GeneratedPropTable({
         // Replace "Ref" with "React.Ref" to match docs convention
         // Includes `<` to avoid picking up `HTMLDivElement` and similar
         .replace(/Ref</g, 'React.Ref<');
-
```

Comment:
nit: leaving whitespace like this is nice, makes the code easier to read

Reply:
DONE